app/saml.py

```diff
@@ -105,7 +105,8 @@ async def sso(request):
   # raise HTTPFound(sso_built_url)
 
   # If we need to redirect to a specific URL:
-  return_to = '{}://{}/index'.format(request.scheme, request.host)
+  return_to = '{}://{}/microservices/index'.format(request.scheme,
```

Comment:
If you're using f-strings elsewhere, you should probably use an f-string here for consistency.

Reply:
Done